main.go

```diff
@@ -4,40 +4,25 @@ package main
 import (
 	"firecontroller/app"
 	"firecontroller/nodecluster"
-	"flag"
 	"fmt"
 	"math/rand"
 	"net"
-	"os"
 	"time"
 
 	"github.com/labstack/echo"
+	"github.com/spf13/viper"
 )
 
 /* The entry point for our System */
 func main() {
-	/* Parse the provided parameters on command line */
-	// makeMasterOnError := flag.Bool(
-	// 	"makeMasterOnError",
-	// 	false,
-	// 	"make this node master if unable to connect to the cluster ip provided.")
-	clusterip := flag.String(
-		"clusterip",
-		"127.0.0.1:8001",
-		"ip address of any node to connnect")
-	myport := flag.String(
-		"myport",
-		"8001",
-		"ip address to run this node on. default is 8001.")
-	flag.Parse()
-
-	makeMasterOnError, exists := os.LookupEnv("DEFAULT_MASTER")
-
-	if !exists {
-		makeMasterOnError = "FALSE"
-	}
-
-	/* Generate id for myself */
+	/* Load Config from Env Vars */
+	configureEnvironment()
+	defaultMaster := viper.GetString("DEFAULT_MASTER")
+	port := viper.GetString("PORT")
```

Comment:
Do you think "PORT" is a little too generic to use as an ENV_VAR? Maybe we can use something like "GF_PORT"?? 
(GF = GoFire or GlutenFree)

Reply:
Good catch, I'm going to merge this and I'll move to something more specific in my next PR